contracts/strategies/traderJoe/strategy-joe-usdce-ust.sol

```diff
@@ -0,0 +1,181 @@
+    address public ust = 0x260Bbf5698121EB85e7a74f2E45E16Ce762EbE11;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyJoeRushFarmBase(
+            usdce_ust_poolId,
+            joe_usdce_ust_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    // **** State Mutations ****
+
+    function harvest() public override onlyBenevolent {
+        // Collects Token Fees
+        IMasterChefJoeV2(masterChefJoeV3).deposit(poolId, 0);
+
+        // Take Avax Rewards    
+        uint256 _avax = address(this).balance;              // get balance of native AVAX
+        if (_avax > 0) {                                    // wrap AVAX into ERC20
+            WAVAX(wavax).deposit{value: _avax}();
+        }
+        
+        // 10% is sent to treasury
+        uint256 _wavax = IERC20(wavax).balanceOf(address(this));  // get balance of WAVAX
+        uint256 _usdce = IERC20(usdce).balanceOf(address(this));    // get balance of USDCe
+        uint256 _ust = IERC20(usdce).balanceOf(address(this));    // get balance of UST  
+        uint256 _joe = IERC20(joe).balanceOf(address(this));      // get balance of JOE 
+        
+        if (_wavax > 0) {
+            uint256 _keep = _wavax.mul(keep).div(keepMax);
+            if (_keep > 0){
+                _takeFeeWavaxToSnob(_keep);
+            }
+            
+            _wavax = IERC20(wavax).balanceOf(address(this));
+        }
+
+        if (_usdce > 0) {
+            uint256 _keep = _usdce.mul(keep).div(keepMax);
+            if (_keep > 0){
+                _takeFeeRewardToSnob(_keep, usdce);
+            }
+            
+            _usdce = IERC20(usdce).balanceOf(address(this));
+        }
+
+        if (_ust > 0) {
+            uint256 _keep = _ust.mul(keep).div(keepMax);
+            if (_keep > 0){
+                _takeFeeRewardToSnob(_keep, ust);
+            }
+            
+            _ust = IERC20(ust).balanceOf(address(this));
+        }
+
+        if (_joe > 0) {
+            
+            uint256 _keep = _joe.mul(keep).div(keepMax);
+            if (_keep > 0) {
+                _takeFeeJoeToSnob(_keep);
+            }
+
+            _joe = IERC20(joe).balanceOf(address(this));
+        }
+
+        // In the case of WAVAX Rewards, swap WAVAX for USDCe and UST
+        if(_wavax > 0){
+            IERC20(wavax).safeApprove(joeRouter, 0);
+            IERC20(wavax).safeApprove(joeRouter, _wavax);   
+            _swapTraderJoe(wavax, usdce, _wavax.div(2));
+            _swapTraderJoe(wavax, ust, _wavax.div(2));
+        }
+
+        // In the case of USDCe Rewards, swap half USDCe for UST
+        if(_usdce > 0){
+            IERC20(usdce).safeApprove(joeRouter, 0);
+            IERC20(usdce).safeApprove(joeRouter, _usdce.div(2));   
+            _swapTraderJoe(usdce, ust, _usdce.div(2));  
```

Comment:
path might not work

Reply:
Both tokens have avax-<token> pools; after checking the `_swapTraderJoe()` function I think it should work

```
function _swapTraderJoe(
        address _from,
        address _to,
        uint256 _amount
    ) internal {
        require(_to != address(0));

        address[] memory path;

        if (_from == wavax || _to == wavax) {
            path = new address[](2);
            path[0] = _from;
            path[1] = _to;
        } else {
            path = new address[](3);
            path[0] = _from;
            path[1] = wavax;
            path[2] = _to;
        }

        IJoeRouter(joeRouter).swapExactTokensForTokens(
            _amount,
            0,
            path,
            address(this),
            now.add(60)
        );
    }